search-app/docs/api.md

```diff
@@ -3732,6 +3738,7 @@ These parameters will match fields within a service. They are case insensitive b
 
 * name
   * options: pattern, ignore_case
+* type
```

Comment:
Add the options description for `type` parameter.

Reply:
done